File: Frontend/angular-app/src/app/modules/inventory/entities/correctionRequestModel.ts
export interface CorrectionRequestModel {
  id: number;
  status: number;
  statusTitle: string;
  codeVoucherGroupId: number;
  documentId: number | null;
  oldData: string;
  verifierUserId: number;
  payLoad: string;
  apiUrl: string;
  viewUrl: string;
  description: string;
  verifierDescription: string;
  requesterDescription: string;
  createdAt: string;
  modifiedById: number | null;
  modifiedAt: string;
  createdById: number;
  username: string;
}

File: Backend/Microservices/Accounting/Eefa.Accounting.Application.Refactor/UseCases/VouchersHeads/Command/SubmitVoucherHeadCorrectionRequest/CorrectionRequestModel.cs
public class CorrectionRequestModel
{
    public int Id { get; set; } = default!;
    public int Status { get; set; } = default!;
    public int CodeVoucherGroupId { get; set; } = default!;
    public int? AccessPermissionId { get; set; } = default!;
    public int? DocumentId { get; set; }
    public string OldData { get; set; } = default!;
    public int VerifierUserId { get; set; } = default!;
    public string? PayLoad { get; set; }
    public string ApiUrl { get; set; } = default!;
    public string? ViewUrl { get; set; }
    public string? Description { get; set; }
}